backend/src/main/java/com/redhat/cloud/notifications/templates/Rhosak.java

```diff
@@ -0,0 +1,92 @@
+package com.redhat.cloud.notifications.templates;
+
+import com.redhat.cloud.notifications.models.EmailSubscriptionType;
+import io.quarkus.qute.CheckedTemplate;
+import io.quarkus.qute.TemplateInstance;
+
+// Name needs to be "Rhosak" to read templates from resources/templates/Rhosak
+public class Rhosak implements EmailTemplate {
+    private static final String SCHEDULED_UPGRADE = "scheduled_upgrade";
```

Comment:
Are you OK with a dash separator in the application name while the event types names are separated with underscores? This is not what you initially requested so I want to make sure it's fine 😃

Reply:
Oh you addressed that question that while I was reviewing 😄

Reply:
Yes, I saw @theute comment in the JIRA and applied the changes. Good spot as usual by the way.